ledger/sandbox-on-x/src/app/scala/com/daml/ledger/sandbox/MainHocon.scala

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.sandbox
+
+import com.daml.ledger.runner.common.HoconCli
+
+object MainHocon {
+  def main(args: Array[String]): Unit =
+    SandboxOnXRunner.run(HoconCli.loadConfigWithOverrides(SandboxOnXRunner.RunnerName, args))
```

Comment:
By dropping out the `ProgramResource` you've lost the setup of the shutdown hook that allows for clean shutdown.